packages/ui/src/components/Button/Button.tsx

```diff
@@ -59,12 +60,14 @@ export function Button({
             {iconPlacement === 'left' && (
                 <Icon
                     icon={icon}
-                    className={twMerge(isIconOnly && (size === 'small' ? `w-4 h-4` : `w-6 h-6`))}
+                    className={twMerge(
+                        !children && variation !== 'navigation' ? 'w-4 h-6' : 'w-6 h-6',
```

Comment:
Shouldn't it be `w-4 h-4` ?
```suggestion
                        !children && variation !== 'navigation' ? 'w-4 h-4' : 'w-6 h-6',
```

Reply:
It's `w-4 h-6` to make it a square button because of the padding on the container

Reply:
Doesn't the zero-width space serve this purpose? It was added there exactly for this reason 🤔

Reply:
Yeah but button has padding `12px 16px` so using rounded values for height and width gives a rectangular button instead of square button

Reply:
Oh, right. Thanks for clearing this one up! 👍🏻

Reply:
Added comment